src/popup/router/pages/Names/AuctionList.vue

```diff
@@ -1,25 +1,29 @@
 <template>
   <div class="auction-list">
     <Filters
-      v-if="activeAuctions.length || auctions.length || loading"
-      v-model="displayMode"
+      v-if="hasAuctions || loading"
+      :value="displayMode"
       :filters="filters"
+      @input="handleFilter"
```

Comment:
alternatively, you can keep `v-model` and to use `watch: { displayMode: { handler, immediate: true } }`

Reply:
I thought it will be clearer and more efficient to have a method here since it is changed only on input event in `Filters.vue`